operator/hack/common.sh

```diff
@@ -126,19 +126,19 @@ function approve_install_plan() {
 
   local install_plan_name
   # `local` ignores `errexit` so we assign value separately: http://mywiki.wooledge.org/BashFAQ/105
-  install_plan_name=$(kubectl get -n "${operator_ns}" subscription.operators.coreos.com stackrox-operator-test-subscription -o jsonpath="{.status.installPlanRef.name}")
+  install_plan_name=$("${ROOT_DIR}/operator/hack/retry-kubectl.sh" < /dev/null get -n "${operator_ns}" subscription.operators.coreos.com stackrox-operator-test-subscription -o jsonpath="{.status.installPlanRef.name}")
   readonly install_plan_name
 
   log "Approving install plan ${install_plan_name}"
-  retry 3 5 kubectl -n "${operator_ns}" patch installplan "${install_plan_name}" --type merge -p '{"spec":{"approved":true}}'
+  retry 3 5 "${ROOT_DIR}/operator/hack/retry-kubectl.sh" < /dev/null -n "${operator_ns}" patch installplan "${install_plan_name}" --type merge -p '{"spec":{"approved":true}}'
 }
 
 function nurse_deployment_until_available() {
   local -r operator_ns="$1"
   local -r version_tag="$2"
 
   log "Patching image pull secret into ${version_tag} CSV..."
-  retry 30 10 kubectl -n "${operator_ns}" patch clusterserviceversions.operators.coreos.com \
+  retry 30 10 "${ROOT_DIR}/operator/hack/retry-kubectl.sh" < /dev/null -n "${operator_ns}" patch clusterserviceversions.operators.coreos.com \
```

Comment:
30 * 10 * (I think a max of 55 seconds delay from the script) could add up to a lot of time

Reply:
Yes, but the script will only retry in very specific circumstances, so I think this will be fine.